src/sonic-config-engine/tests/sample-arista-7260-t0-minigraph.xml

```diff
@@ -0,0 +1,2869 @@
+<DeviceMiniGraph xmlns="Microsoft.Search.Autopilot.Evolution" xmlns:i="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
can you model Arista-7060CX-32S-D48C8 instead? That is the hwsku that connects to EdgeZoneAggregator

Reply:
Ok